examples/flava/requirements.txt

```diff
@@ -6,3 +6,4 @@ DALL-E==0.1
 omegaconf==2.1.2
 hydra-core==1.1.2
 transformers==4.16.0
+pycocotools==2.0.4
```

Comment:
do u need this?

Reply:
yes I couldn't run the script on the cluster because I didn't have this package